packages/zoe/src/contractSupport/topics.js

```diff
@@ -17,7 +17,7 @@ export const PublicTopicShape = M.splitRecord(
  * @typedef {{
  *   description?: string,
  *   subscriber: Subscriber<T>,
- *   storagePath: ERef<string>,
+ *   storagePath: ERef<string> | string,
```

Comment:
remove. `ERef<T>` is already `PromiseLike<T> | T` https://endojs.github.io/endo/types/_endo_eventual_send.ERef.html